docs/reference/experiment-config.txt

```diff
@@ -229,8 +229,7 @@ created with the Determined CLI. It may contain the following fields:
     - ``smaller_is_better``: Specifies whether to minimize or
       maximize the metric defined above. The default value is true
       (minimize).
-    - ``max_steps``: Specifies how many steps to allocate to the
-      trial.
+    - ``max_length``: Specifies how many long to train the trial.
```

Comment:
Here and below, can we include more detail on how this value is specified? e.g., units, structure of the nested struct.